### PR TITLE
fix: APP-2603 - Implemented proposals created stat

### PR DESCRIPTION
**Pull request by babarella**
## Description

- Implemented `proposals created` stat
- Created hook for fetched creators proposal (will be needed for APP-2607)
- Improved grid view of the stats to support display of odd total number of stats

Task: [APP-2603](https://aragonassociation.atlassian.net/browse/APP-2603)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2603]: https://aragonassociation.atlassian.net/browse/APP-2603?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/daoMember.tsx [hunk 1]
```diff
@@ -80,6 +81,18 @@ export const DaoMember: React.FC = () => {
...
+  const {
+    data: memberCreatedProposals,
+    isLoading: isMemberCreatedProposalsLoading,
+  } = useCreatorProposals(
+    {
+      address: memberAddress,
+      pluginAddress: pluginAddress || '',
+      pluginType,
+    },
+    {enabled: !!memberAddress && !!daoDetails}
```
Comment (cgero-eth) [inline src/pages/daoMember.tsx]: In that way we avoid calling the hook with an empty string on the `pluginAddress` parameter which could throw some error on the console
```suggestion
      pluginAddress: pluginAddress as string,
      pluginType,
    },
    {enabled: !!memberAddress && !!daoDetails && !!pluginAddress}
```

File: src/services/aragon-sdk/aragon-sdk-service.api.ts [hunk 2]
```diff
@@ -51,6 +51,13 @@ export interface IFetchMemberParams {
   blockNumber?: number;
 }
 
+export interface IFetchCreatorProposalsParams {
+  pluginAddress: string;
+  address: string;
+  pluginType: PluginTypes | undefined;
```
Comment (cgero-eth) [inline src/services/aragon-sdk/aragon-sdk-service.api.ts]: Just to follow how it is currently defined on other param types 
```suggestion
  pluginType?: PluginTypes;
```

File: src/services/aragon-sdk/queries/use-member.ts [hunk 3]
```diff
@@ -80,13 +80,15 @@ export const useMember = (
   options: UseQueryOptions<TokenVotingMember> = {}
 ) => {
   const client = usePluginClient('token-voting.plugin.dao.eth');
-  const {address, network} = useWallet();
+  const {address} = useWallet();
+  const {network} = useNetwork();
+
   const baseParams = {
     address: address as string,
```
Comment (cgero-eth) [inline src/services/aragon-sdk/queries/use-member.ts]: We don't probably need the current user address to build the query key, I would suggest to remove the `useWallet` hook here:
```suggestion
  const {network} = useNetwork();

  const baseParams = {
```
And update the `getMember` query key params to omit the `address` parameter as follow:
```
getMember: (
    baseParams: Omit<IAragonSdkBaseParams, 'address'>,
    params: IFetchMemberParams
```

File: src/services/aragon-sdk/queries/use-creator-proposals.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  options: UseQueryOptions<ProposalBase[]> = {}
+) => {
+  const client = usePluginClient(params.pluginType);
+  const {address} = useWallet();
+  const {network} = useNetwork();
+
+  const baseParams = {
+    address: address as string,
+    network: network,
+  };
```
Comment (cgero-eth) [inline src/services/aragon-sdk/queries/use-creator-proposals.ts]: Same here, I think we can safely remove the `address` parameter for the react-query key:
```suggestion
  const {network} = useNetwork();

  const baseParams = {
    network: network,
  };
```

**Review by cgero-eth — COMMENTED** [review]
Nice work @babarella! I left some minor comments

File: src/services/aragon-sdk/queries/use-creator-proposals.ts (hunk 4 above)
Comment (Barukimang) [reply src/services/aragon-sdk/queries/use-creator-proposals.ts]: Have confirmed the fixes are working @RuggeroCino , see test results here: https://aragonassociation.atlassian.net/browse/APP-2603?focusedCommentId=20166

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**